Overview.src.html

```diff
@@ -3965,9 +3962,63 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 is itself associated with <span title=concept-Response-response>response</span>'s
 <span title=concept-response-header-list>header list</span>.
 
-<p>A <code>Response</code> object's <span title=concept-Body-body>body</span> is its
-<span title=concept-Response-response>response</span>'s
-<span title=concept-response-body>body</span>.
+<p>A <code>Response</code> object also has an associated
+<dfn title=concept-Response-readable-stream>readable stream</dfn> of type
+<span title=concept-ReadableStream>ReadableStream</span> which is initially null.
+
+<p class = "note no-backref">The type of the associated
+<span title=concept-Response-readable-stream>readable stream</span> is defined as
+<span title=concept-ReadableStream>ReadableStream</span>, but it is currently discussed. Depending
+on the discussion, the type might be changed to <code>ReadableByteStream</code>. Hence the type name
+is not strictly specified at this moment, though the behavior is specified because
+<code>ReadableByteStream</code> has all methods and properties which
+<span title=concept-ReadableStream>ReadableStream</span> has.
+
+<p>A <code>Response</code> object also has an associated
+<dfn title=concept-Response-locked>locked</dfn> predicate that returns true if the associated
+<span title=concept-Response-readable-stream>readable stream</span> is not null and it is
+<span title=concept-ReadableStream-locked>locked</span>.
+
+<p>A <code>Response</code> object's <span title=concept-Body-disturbed>disturbed</span> predicate
+returns true if the associated <span title=concept-Response-readable-stream>readable stream</span>
+is not null and it is <span title=concept-ReadableStream-disturbed>disturbed</span>.
+
+<p>A <code>Response</code> object's <span title=concept-body-consume-body>consume body</span>
+algorithm, which given a <var title>type</var>, runs these steps:
+
+<ol>
+ <li><p>If this <code>Response</code> is <span title=concept-Body-disturbed>disturbed</span> or
+ <span title=concept-Response-locked>locked</span>, return a new promise rejected with a
+ <code>TypeError</code>.
+
+ <li><p>Let <var title>stream</var> be <span title=concept-Response-readable-stream>readable
+ stream</span>.
+
+ <li><p>If <var title>stream</var> is null, set <var title>stream</var> to an
+ <span title=concept-empty-ReadableStream>empty ReadableStream</span>.
+
+ <li><p>Let <var title>reader</var> be the result of <span title=concept-get-reader>getting</span>
+ a reader from <var title>stream</var>. If that threw an exception, return a new promise rejected
+ with that exception.
+
+ <li><p>Let <var title>p</var> be a new promise.
+
+ <li><p>Let <var title>promise</var> be the result of
```

Comment:
I think this section would be clearer if instead of p and promise, you just said "Let promise be the result of reading all bytes from stream with reader. Return the result of transforming promise by a fulfillment handler that ...". It translates more naturally to idiomatic promise code that way too.